### PR TITLE
Small/big science update

### PR DESCRIPTION
**Pull request by Guidesu**
Purple asked me to do voting in the discord.
Casey#4535 in the discord suggested Senior Researcher be removed (nuked).
She got several positives feedback and suggested adding o-3 and o-4 to normal scientist which i did.

What i mostly modified was:
Removing SR from code.
Fixing Jobs having EC instead of the fleet. Because what i was told EC was nuked several months ago.
Adding O-1 , O-2 , O-3, O-4 to scientist. Or just adding O-3 + O-4.
Removing SMC Warrant Officer from roboticist and adding an fleet one.
Giving researcher assistant e-2 to e-4.
And adding two alt titles to compensate for the SR removal: Researcher Advisor and Senior Researcher.

If Loorey or Purple wishes to correct anything the code just do it! :)
Just don't let me wait for 3 days.
If you wish to discuss in discord just ping me.
Guidesu#7987

### CODE REVIEW

**Comment by gilvian** [comment]
Science main here.

i will miss my EC stuff :(

Removing SMC from science seems okay

Dont give science assistant more ranks, they are supposed to be assistants, not have seniority ranks that corpsman can get, atleast max E4.

If we are removing senior role we dont need it as alt title, its fine with science having those 2 roles cut since nobody played them, or they tried to be big dick mcgees


Overall okay PR after those changes.

**Comment by Guidesu** [comment]
Done!

File: maps/torch/job/research_jobs.dm [hunk 1]
```diff
@@ -1,42 +1,3 @@
...
-	                    SKILL_DEVICES     = SKILL_ADEPT,
-	                    SKILL_SCIENCE     = SKILL_EXPERT)
-
-	max_skill = list(   SKILL_ANATOMY     = SKILL_EXPERT,
-	                    SKILL_DEVICES     = SKILL_EXPERT,
-	                    SKILL_SCIENCE     = SKILL_EXPERT)
-	skill_points = 24
 
 /datum/job/scientist
 	title = "Scientist"
```
Comment (io-util) [inline maps/torch/job/research_jobs.dm]: Delete the leading empty line here.

File: maps/torch/job/research_jobs.dm [hunk 2]
```diff
@@ -53,7 +14,7 @@
 		"Xenobiologist",
 		"Xenobotanist",
 		"Circuitry Designer",
-		"Chemical Researcher"
+		"Chemical Researcher",
 	)
```
Comment (io-util) [inline maps/torch/job/research_jobs.dm]: Redundant Comma.

```suggestion
		"Chemical Researcher"
	)
```

**Review by io-util — CHANGES_REQUESTED** [review]

File: maps/torch/job/research_jobs.dm (hunk 1 above)
Comment (Guidesu) [reply maps/torch/job/research_jobs.dm]: deleted

**Comment by Guidesu** [comment]
fixed

File: maps/torch/job/research_jobs.dm [hunk 3]
```diff
@@ -160,13 +123,16 @@
...
 		/datum/mil_branch/solgov
 	)
 	allowed_ranks = list(
-		/datum/mil_rank/ec/e3,
-		/datum/mil_rank/ec/e5,
+		/datum/mil_rank/fleet/e2,
+		/datum/mil_rank/fleet/e3,
+		/datum/mil_rank/fleet/e4,
+		/datum/mil_rank/fleet/e5,
+		/datum/mil_rank/fleet/e6,
```
Comment (Firefox13) [inline maps/torch/job/research_jobs.dm]: изображение
this isn't e-4

**Comment by io-util** [comment]
Closing this as we are not reviewing / accepting PR's on our old repo anymore.
New Repo: https://github.com/VestaOfOrion/Vesta.Bay/

**Pull request closed without merging**